LinearElasticity_jll does not ship an import library on Windows, so look for
the DLL itself instead of `libElas.dll.a`: MinGW's linker can link directly
against a DLL.

--- a/cmake/modules/FindElas.cmake
+++ b/cmake/modules/FindElas.cmake
@@ -39,13 +39,14 @@ FIND_PATH(ELAS_INCLUDE_DIR
 
 # Check for elas library (search in common locations in order to prevent
 # modifications in ELAS installation)
-FIND_LIBRARY(ELAS_LIBRARY
-  NAMES Elas Elas${ELAS_LIB_SUFFIX}
+FIND_FILE(ELAS_LIBRARY
+  NAMES libElas.dll
   HINTS ${ELAS_LIBRARY}
   $ENV{ELAS_LIBRARY}
   $ENV{HOME}/lib
   ${ELAS_DIR}/lib
   $ENV{ELAS_DIR}/lib
+  PATH_SUFFIXES bin ../bin lib ../lib
   DOC "The ELAS library"
   )
 
